File: app/src/main/java/com/prashant/masterbuddy/ws/model/RegisterResult.java
package com.prashant.masterbuddy.ws.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by tanmay.agnihotri on 5/1/18.
 */

public class RegisterResult {

    @SerializedName("response")
    public Response response;

    public static class Response{
        @SerializedName("message")
        private String message;

        @SerializedName("isSuccessfullyRegistred")
        private boolean isSuccessfullyRegistred;

        @SerializedName("userID")
        private int userID;

        public String getMessage() {
            return message;
        }

        public boolean isSuccessfullyRegistred() {
            return isSuccessfullyRegistred;
        }

        public int isUserID() {
            return userID;
        }
    }
}

File: app/src/main/java/com/prashant/masterbuddy/ws/model/RegisterResult.kt
package com.prashant.masterbuddy.ws.model

import com.google.gson.annotations.SerializedName

/**
 * Created by tanmay.agnihotri on 5/1/18.
 */

class RegisterResult {

    @SerializedName("response")
    var response: Response? = null

    class Response {
        @SerializedName("message")
        val message: String? = null

        @SerializedName("isSuccessfullyRegistred")
        val isSuccessfullyRegistred: Boolean = false

        @SerializedName("userID")
        val isUserID: Int = 0
    }
}
